src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawPanel2Job2d.java

```diff
@@ -27,7 +29,10 @@ public void operateTo(int x, int y) {
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
 
-		drawLine(line);
+		this.startX = line.getEndCoordinateX();
+		this.startY = line.getEndCoordinateY();
+
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.